alerta-covid/db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_04_08_184023) do
+ActiveRecord::Schema.define(version: 2021_03_30_055117) do
```

Comment:
Just wonder what happened here?

Is this actually changing to the latest schema version?

Did you try to do `git pull` in your branch?

Reply:
When the issue of some bug fixes was first approved on the master, the PR had a merge conflict on that line only, so when solving it from the github website, it performed automatic master merge.😅  (I have to practice this part more)